app-text/sigil/sigil-0.8.900_pre-r1.ebuild

```diff
@@ -63,8 +63,9 @@ src_configure() {
 	cmake-utils_src_configure
 
 	cd "${WORKDIR}/${P}_build" || die "Where is the build dir?"
-	for i in CMakeCache.txt src/CMakeFiles/sigil.dir/link.txt $(find . -name '*.make')
+	for i in $(grep -Rl "O3 -DNDEBUG" .)
```

Comment:
Oh, I guess that's because of the CMAKE_BUILD_TYPE override. I wonder if that's good for anything.